index.html

```diff
@@ -50,6 +50,189 @@ <h1 class="intro height">I’m Kartikay Glad to see you!</h1>
+              <p class="para-two">2015</p>
+            </div>
+            <p class="card-body">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="tech-stack">
+              <li><button class="tech-items" type="button">html</button></li>
+              <li><button class="tech-items" type="button">css</button></li>
+              <li><button class="tech-items" type="button">javascript</button></li>
+            </ul>
+            <a class="card-button" href="#">See Project</a>
+          </div>
+        </div>
+        
+        <div class="card">
+          <div class="card-element">
+            <div class="img-container">
+              <img class="work-image" src="./Images/Tonic-two.png" alt="work-Image" />
+            </div>
+            <h2 class="card-title">Tonic</h2>
+            <div class="card-section">
+              <p class="para-one">CANOPY</p>
+              <i class="fa fa-circle"></i>
+              <p class="para-two">Back End Dev</p>
+              <i class="fa fa-circle"></i>
+              <p class="para-two">2015</p>
+            </div>
+            <p class="card-body">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="tech-stack">
+              <li><button class="tech-items" type="button">html</button></li>
+              <li><button class="tech-items" type="button">css</button></li>
+              <li><button class="tech-items" type="button">javascript</button></li>
+            </ul>
+            <a class="card-button" href="#">See Project</a>
+          </div>
+        </div>
+        
+        <div class="card multipost">
+  
+          <div class="card-element">
+            <div class="img-container multipost">
+              <img class="work-image" src="./Images/Multiple-two.png" alt="work-Image" />
+            </div>
+            <h2 class="card-title">
+              Multi-Post Stories
+            </h2>
+            <div class="card-section">
+              <p class="para-one">CANOPY</p>
+              <i class="fa fa-circle"></i>
+              <p class="para-two">Back End Dev</p>
+              <i class="fa fa-circle"></i>
+              <p class="para-two">2015</p>
+            </div>
+            <p class="card-body">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="tech-stack">
+              <li><button class="tech-items" type="button">html</button></li>
+              <li><button class="tech-items" type="button">css</button></li>
+              <li><button class="tech-items" type="button">javascript</button></li>
+            </ul>
+            <a class="card-button" href="#">See Project</a>
+          </div>
+        </div>
+  
+      </div>
+      <div class="about-sec">
```

Comment:
- [ ] As I pointed you in a previous comment, the following section needs to be wrapped in a `section` tag

Reply:
Thank you for the suggestion. I have wrapped it in a section tag.